react-app/webpack.config.babel.js

```diff
@@ -0,0 +1,67 @@
+import webpack from 'webpack';
+import path from 'path';
+import HtmlWebpackPlugin from 'html-webpack-plugin';
+import MiniCssExtractPlugin from 'mini-css-extract-plugin';
+
+module.exports = {
+  entry: ['./src/app.js'],
```

Comment:
Try to add a root path(parent folder) here, which you can use also for later purposes in webpack.